example/ck_tile/01_fmha/README.md

```diff
@@ -26,50 +26,63 @@ To speed up compile time, we instantiate the kernels into separate file. In this
+            -squant    if using static quantization fusion or not. auto: fp8 will default use squant, other will not (default:auto)
+                       0: no static quant(not implemented) 1: apply scale_p and scale_o with respect to P and O.
+                       calculate scale_s, scale_p, scale_o according to range_q, range_k, range_v, range_p, range_o
+             -iperm    permute input (default:1)
+                       if true, will be b*h*s*d, else b*s*h*d
+             -operm    permute output (default:1)
+              -bias    n or 0, no bias (default:n)
+                       e(lementwise) or 1, elementwise bias with 1*1*s*s. e:1, 1*h*s*s. e:2, b*h*s*s
+                       a(libi) or 2, alibi with 1*h. a:1, b*h
+              -prec    data type. fp16/bf16/fp8/bf8 (default:fp16)
+              -mask    0: no mask, 1: top-left(same as 't'), 2:bottom-right(same as 'b') (default:0)
+                       't', top-left causal mask, 'b', bottom-r causal mask
+                       't:l,r', top-left sliding window attn(swa) with FA style left right size
+                       'b:l,r', bottom-r sliding window attn(swa) with FA style left right size
+                       'xt:window_size', xformer style masking from top-left, window_size negative is causal, positive is swa
+                       'xb:window_size', xformer style masking from bottom-r, window_size negative is causal, positive is swa
+                       'g:y,x', generic attention mask coordinate with y/x size (only debug purpose for now)
+           -vlayout    r for row-major(seqlen*hdim), c for col-major(hdim*seqlen) (default:r)
+               -lse    0 not store lse, 1 store lse (default:0)
+             -kname    if set to 1 will print kernel name (default:0)
+              -init    init method. ui, uniform random int, ni, normalized random int (default:uf)
+                       uf, uniform random float, nf, normalized random float, tf, trig float, uf:q, quantization
+              -seed    random seed used for initializing input tensors. 0 for non-deterministic seed (default:11939)
+            -p_drop    0~1 probability of dropout (default:0)
+         -drop_seed    seed for random number generator (default:1)
+       -drop_offset    offset for random number generator (default:0)
+             -timer    gpu:gpu timer, cpu:cpu timer (default:gpu)
+        -rotary_dim    RoPE rotary dimension. rotary_dim <= 0 means not apply RoPE at all (default:0)
+-rotary_interleaved    whether to apply interleaved RoPE (default:1)
+        -num_splits    # of splits for key/value. 0 to determine actual number by heuristic (default:1)
+   -page_block_size    paged-kvcache block size. 0 means not use paged-kvcahe (default:0)
+   -cache_batch_idx    whether to use index map to the kvcache (default:0)
+            -warmup    number of iterations before benchmark the kernel (default:5)
+            -repeat    number of iterations to benchmark the kernel (default:20)
```

Comment:
I realize that this is the output of the output. I suggest that if it's subject to change, to not hardcode it in from a maintenance perspective. I think simply saying "this will provide a list of all the arguments" would be fine (without the "supported").

(Also, the descriptions hardcoded in the code itself need to be fixed, but that's a different issue.)

Reply:
Thanks. I will leave the existing arguments there and only update the descriptions